functions/wait/state/activeNodes.go

```diff
@@ -18,10 +18,15 @@ func ActiveNodes(t *testing.T, client *rancher.Client, clusterID string) (done b
 	})
 	require.NoError(t, err)
 
+	if err != nil {
```

Comment:
Small nitpick, would it be a good idea to make this variable a little more descriptive? i.e list_err

Reply:
updated